### PR TITLE
[Security] Add a little explanations in supports() description

### PR DESCRIPTION
**Pull request by Thaldos**
| Q             | A
| ------------- | ---
| Branch?       | 5.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | -
| License       | MIT
| Doc PR        |  -

It is a very little modification in comment, just to help to understand quickly how the supports() works.

### CODE REVIEW

**Comment by OskarStark** [comment]
CleanShot 2021-08-07 at 11 01 51@2x

It looks like your commiter email is not associated with your Github Account.

**Comment by Thaldos** [comment]
> It looks like your commiter email is not associated with your Github Account.

Ah yeah :/. "vincent.gode@groupe-celios.fr" is not my GitHub email address but it is me. It is one of my others email address. 

Have I to fix it or can I let it as is? (If it is ok  for you, it is ok for me too.)

Thx

File: src/Symfony/Component/Security/Http/Authenticator/AuthenticatorInterface.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ interface AuthenticatorInterface
     /**
      * Does the authenticator support the given Request?
      *
-     * If this returns false, the authenticator will be skipped.
+     * If this returns true authenticate() will be called. If false, the authenticator will be skipped.
```
Comment (chalasr) [inline src/Symfony/Component/Security/Http/Authenticator/AuthenticatorInterface.php]:
```suggestion
     * If this returns true, authenticate() will be called. If false, the authenticator will be skipped.
```

**Comment by OskarStark** [comment]
> Ah yeah :/. "vincent.gode@groupe-celios.fr" is not my GitHub email address but it is me. It is one of my others email address.
>
> Have I to fix it or can I let it as is? (If it is ok for you, it is ok for me too.)
>
> Thx

It is not mandatory, but would be helpful. You can add the email to your Github account:
https://github.com/settings/emails

**Comment by Thaldos** [comment]
> It is not mandatory, but would be helpful. You can add the email to your Github account:
> https://github.com/settings/emails

Done, thanks :)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Thaldos.

**Pull request merged**